src/common/schemas/specifications/suspended-payment-cancellation.json

```diff
@@ -9,11 +9,11 @@
       "$ref": "address",
       "description": "The address of the owner of the suspended payment to cancel."
     },
-    "paymentSequence": {
+    "suspensionSequence": {
```

Comment:
I think the description clarifies it enough; we use the convention of "sequence" everywhere for "account sequence number".